### PR TITLE
Release notes for v1.19.0-rc1

### PR DESCRIPTION
**Pull request by shubham-bansal96**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -110,7 +110,7 @@ algolia_docsearch = true
 # current release branch - could be rc
 release_branch = "main"
 # the main version. Never is rc.
-release_version = "v1.18.0"
```
Comment (sanketsudake) [inline config.toml:113]: keep this 1.18 only We wll change this we release 1.19.

File: content/en/docs/releases/v1.19.0-rc1.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+title: "v1.19.0-rc1"
+linkTitle: v1.19.0-rc1
+weight: 78
+---
+
+## Features
```
Comment (sanketsudake) [inline content/en/docs/releases/v1.19.0-rc1.md:7]: need better formatting for this.

File: config.toml (hunk 1 above)
Comment (shubham-bansal96) [reply config.toml]: done

File: netlify.toml [hunk 3]
```diff
@@ -518,8 +518,8 @@ status = 301
 
 [[redirects]]
 force = true
-from = "/docs/releases/v1.18.0-rc2/"
-to = "/docs/releases/v1.18.0/"
+from = "/docs/releases/v1.18.0/"
```
[the remark this answers is not in the document]
Comment (shubham-bansal96) [reply netlify.toml]: reverted this change.

File: content/en/docs/releases/v1.19.0-rc1.md (hunk 2 above)
Comment (shubham-bansal96) [reply content/en/docs/releases/v1.19.0-rc1.md:7]: done.

**Review by sanketsudake — APPROVED** [review]

**Pull request merged**